lib/ansible/modules/windows/win_domain_membership.ps1

```diff
@@ -22,6 +20,9 @@ Function Write-DebugLog {
 
     Write-Debug $msg
 
+    $log_path = $null
```

Comment:
Shouldn't be needed.

Reply:
powershell best practice to init vars before assigning to them